src/robot/running/handlerstore.py

```diff
@@ -65,5 +65,5 @@ def _raise_no_single_match(self, name, found):
                             % (where, name))
         error = ["%s contains multiple keywords matching name '%s':"
                  % (where, name)]
-        names = sorted(handler.orig_name for handler in found)
+        names = seq2str([getattr(f, 'orig_name', None) or f.name for f in found])
```

Comment:
If `f.name` always exists, could just pass it as the default value for `getattr`.